<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_MainWindow</class>
 <widget class="QMainWindow" name="qt_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>象棋</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_single">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>遊戲開始</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multi">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>多人遊戲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnt_setting">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>設定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_end">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>遊戲結束</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>象棋</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vs_project.qrc"/>
 </resources>
 <connections/>
</ui>
